.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: ci testing
+
+
+on:
+  push:
+    branches: master
+  pull_request:
+
```

Comment:
Is it expected that this section is empty?

Reply:
Yes, it means "on any pull request", afaiu.